aep/general/0159/aep.md.j2

```diff
@@ -37,6 +37,10 @@ guidance in [unreachable resources].
   specified with `*` and permit the collection to be specified; a URI pattern
   **must not** hard-code the `-` character.
 
+{% tab oas %}
```

Comment:
is there a way we can ensure site-generator-beta builds properly to catch issues like this? would be a good e2e test.